docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -75,25 +75,36 @@ only the termination message:
 {% raw %}  kubectl get pod termination-demo -o go-template="{{range .status.containerStatuses}}{{.lastState.terminated.message}}{{end}}"{% endraw %}
 ```
 
-## Setting the termination log file
-
-By default Kubernetes retrieves termination messages from
-`/dev/termination-log`. To change this to a different file,
-specify a `terminationMessagePath` field for your Container.
-
-For example, suppose your Container writes termination messages to
-`/tmp/my-log`, and you want Kubernetes to retrieve those messages.
-Set `terminationMessagePath` as shown here:
-
-    apiVersion: v1
-    kind: Pod
-    metadata:
-      name: msg-path-demo
-    spec:
-      containers:
-      - name: msg-path-demo-container
-        image: debian
-        terminationMessagePath: "/tmp/my-log"
+## Customizing the termination message
+
+By default Kubernetes retrieves termination messages from the termination message file
+specified in the `terminationMessagePath` field of a Container. The default file
+used is `/dev/termination-log`. By customizing this field you can tell Kubernetes
+to use a different file. Kubernetes use the contents from the specified file to
+populate the container's status message on both success and failure.
+
```

Comment:
Suggestion:
Kubernetes retrieves termination messages from the file specified in the `terminationMessagePath`field, which has a default value of `/dev/termination-log`. By customizing this field, you can tell Kubernetes to use a different file. Kubernetes uses the contents of the specified file to populate the Container's status message on both success and failure.

In the following example, the container writes termination messages to
`/tmp/my-log` for Kubernetes to retrieve:

 ```yaml
...
```
Moreover, users can set the `terminationMessagePolicy` field
...
2048 bytes or 80 lines, whichever is smaller.